sdk/main/src/Transaction.cc

```diff
@@ -852,7 +922,11 @@ void Transaction<SdkRequestType>::addTransaction(const proto::Transaction& trans
 template<typename SdkRequestType>
 void Transaction<SdkRequestType>::addTransaction(const proto::SignedTransaction& transaction)
 {
-  mImpl->mTransactions.push_back(proto::Transaction());
+  proto::Transaction tx;
+  tx.ParseFromString(transaction.bodybytes());
```

Comment:
Can you create a `proto::Transaction` successfully from a serialized `proto::SignedTransaction`? I would think here it needs to be
```
proto::Transaction tx;
tx.signedtransactionbytes = transaction.SerializeAsString();
```